dev/bots/analyze.dart

```diff
@@ -472,6 +473,32 @@ Future<void> _verifyNoBadImportsInFlutterTools(String workingDirectory) async {
   }
 }
 
+final RegExp _licensePattern = RegExp(r'''Copyright \d{4}''');
```

Comment:
Can this just use single quotes `'` instead of `'''`?

Reply:
Yes.